app/views/crime_applications/_current_assignment.html.erb

```diff
@@ -1,29 +1,32 @@
 <dl class="govuk-summary-list">
-  <dt class="govuk-summary-list__key">
-    <%= t(:assigned_to, scope: 'crime_applications.labels') %>
-  </dt>
-
-  <% if crime_application.current_assignment.nil? %>
-    <dd class="govuk-summary-list__value">
-      <%= 'Unassigned' %>
-    </dd>
-    <%= button_to t('calls_to_action.assign_to_self'),
-                  assigned_applications_path(
-                    crime_application_id: crime_application.id
-                  ),
-                  method: :post,
-                  class: 'govuk-button govuk-button--secondary govuk-!-margin-top-1' %>
-  <% else %>
-    <dd class="govuk-summary-list__value">
-      <%= crime_application.current_assignment.user.name %>
-    </dd>
-
-    <dd class="govuk-summary-list__action">
-    <% if crime_application.current_assignment.user_id == current_user_id %>
-    <%= button_to t('calls_to_action.unassign_from_self'),
-                  assigned_application_path(crime_application),
-                  method: :delete,
-                  class: 'govuk-button govuk-button--secondary' %>
+  <dd class="govuk-summary-list__value govuk-!-width-one-half govuk-!-padding-0 govuk-body-l">
+    <div class="govuk-row">
+      <strong><%= t(:assigned_to, scope: 'crime_applications.labels') %></strong>
+      <% if crime_application.current_assignment.nil? %>
+        <%= 'Unassigned' %>
```

Comment:
Not sure this needs to be in an erb <%= as it is a string literal

Reply:
Yep, meant to change that - that's how I found it

Reply:
Wanted to put it into en.local